comps/comps-foreman-el9.xml

```diff
@@ -62,10 +62,10 @@
       <packagereq type="default">nodejs-datatables.net</packagereq>
       <packagereq type="default">nodejs-datatables.net-bs</packagereq>
       <packagereq type="default">nodejs-datatables.net-dt</packagereq>
+      <packagereq type="default">nodejs-datatables.net-dt</packagereq>
```

Comment:
This is a duplicate of the line before it.

Reply:
my bad, done.